style/styles.css

```diff
@@ -0,0 +1,3 @@
+.header1{
+    color: aquamarine;
+}
```

Comment:
At Microverse and most likely in your future job, it'll be required for linter checks to pass for a pull request to be considered as valid. kindly ensure that all checks are passed evidenced by a green check (instead of a red cross) beside the last commit.

Passing linter checks:
![image](https://user-images.githubusercontent.com/20554583/204665267-5fefde0c-d4f1-4c72-be1e-94319464caca.png)


Failing Linter checks:
![image](https://user-images.githubusercontent.com/20554583/204665214-6b2a6b2e-96a7-4b56-b9e8-c26d5b9db9e5.png)